README.md

```diff
@@ -68,7 +68,7 @@ By default, the build is targeted against Eclipse Oxygen / 4.7.
 You can explicitly set the `eclipse.target` property to
 `photon` (4.8).
 ```
-$ mvn -Declipse.target=oxygen package
+$ mvn -Declipse.target=photon package
```

Comment:
I noticed this still doesn't work. I think it should be

`$ (cd eclipse ; mvn -Pide-target-platform -Declipse.target=XXX package)`

Right, @briandealwis?

Reply:
This should remain at `eclipse.target=oxygen`.  `photon` doesn't work as it's now the default; Maven doesn't allow an OR-style profile activation unfortunately.

Reply:
I also think you need `-Pide-target-platform` after moving into the `eclipse` sub-directory.

Reply:
Yup, @chanseokoh's right.